src/transformers/pipelines/text_to_audio.py

```diff
@@ -105,13 +128,22 @@ def preprocess(self, text, **kwargs):
         return output
 
     def _forward(self, model_inputs, **kwargs):
+        forward_params = kwargs["forward_params"]
+        generate_kwargs = kwargs["generate_kwargs"]
+
```

Comment:
```suggestion
        kwargs = self._ensure_tensor_on_device(forward_params, device=self.device)
        forward_params = kwargs["forward_params"]
        generate_kwargs = kwargs["generate_kwargs"]
```
we can do this only once in the code then proceed